### ISSUE TITLE
Support <ol>, <ul> and <li>

### ISSUE THREAD

**Issue by twogood** [soundasleep/html2text#22]
For a list like this now:
```html
<ol>
<li>foo</li>
<li>bar</li>
<li>baz</li>
</ol>
```

The text would become
```
foobarbaz
```

But I would at least want it to become
```
foo
bar
baz
```

But even better:

```
1. foo
2. bar
3. baz
```

Similar for ```<ul>```.

Maybe someone already has implemented this?

**Comment by twogood** [comment]
Some support here: https://github.com/utrotzek/html2text/commit/44664ea2ab9aca409e0d92db792638afc8202c4c

**Comment by soundasleep** [comment]
Yes this has been implemented basically in PR #19: https://github.com/soundasleep/html2text/pull/19

(Actually doing numbering is a much more complex issue)
